planter/planter.sh

```diff
@@ -39,19 +39,13 @@ IMAGE="${IMAGE_NAME}:${TAG}"
 # - WORKSPACE is assumed to be in your current git repo, or alternatively $PWD
 REPO=$(git rev-parse --show-toplevel 2>/dev/null || true)
 REPO=${REPO:-${PWD}}
-# - the bazel cache is in $HOME/.cache/bazel
-#
-# - we could just mount $HOME/.cache/bazel, but unfortunately some rules
-# are still not terribly hermetic and actually need $HOME (rules_node)
-#
-# - $HOME/.docker shouldn't be mounted so we clobber it with a tmpfs.
-# You can have a $HOME/.docker/config.json for logging in to docker hub,
-# which on mac will actually store credentials in your keychain which
-# the container cannot access, this breaks rules_docker. (issues/5736)
+# To build kubernetes, first you checkout github.com/kubernetes/kubernetes
```

Comment:
can you put a similar new comment here instead explaining what we mount and why so that this stays self documenting?